src/c_api.cpp

```diff
@@ -1175,6 +1285,35 @@ int LGBM_BoosterPredictForCSR(BoosterHandle handle,
   API_END();
 }
 
+int LGBM_BoosterPredictForCSRSingleRow(BoosterHandle handle,
+                              const void* indptr,
+                              int indptr_type,
+                              const int32_t* indices,
+                              const void* data,
+                              int data_type,
+                              int64_t nindptr,
+                              int64_t nelem,
+                              int64_t,
```

Comment:
what is the variable here?

Reply:
(int64_t,)

Reply:
looks like this is num col, shouldn't it be used here (or removed in signature otherwise)

Reply:
good point. I copied this from the function above. you want to remove it there too?

Reply:
no, if it's copied I guess we should keep it then